app/src/main/java/com/android/unio/model/events/EventTypeColor.kt

```diff
@@ -0,0 +1,24 @@
+package com.android.unio.model.events
+
+import androidx.compose.ui.graphics.Color
+
+// class for event types and their corresponding colors
+data class EventTypeColor(val type: String, val color: Color)
+
+// list of event types with their associated colors
+val eventTypeColors =
+    listOf(
+        EventTypeColor("festival", Color(0xFF6200EE)), // purple
+        EventTypeColor("apéro", Color(0xFF03DAC5)), // teal
+        EventTypeColor("soirée", Color(0xFFFF5722)), // deep orange
```

Comment:
Consider translate "type" in english, for consistency (like "apéro" or "soirée")

Reply:
Thanks !